test/telemetry.spec.ts

```diff
@@ -13,75 +14,137 @@ class MockTelemetryExporter extends MockClass {
 }
 
 describe('DefaultTelemetryProvider', () => {
```

Comment:
@djwhatley-fs I recommend hiding whitespace when reviewing this file. The changes to existing tests reduced nesting.